lib/osf-components/addon/components/addons-service/manager/component.ts

```diff
@@ -52,46 +50,24 @@ export default class AddonsServiceManagerComponent extends Component<Args> {
     @task
     @waitFor
     async getAddonProviders() {
-        const legacyProviders: LegacyProvider[] = await taskFor(this.legacyProviders).perform();
         const serviceStorageProviders: Provider[] = await taskFor(this.serviceStorageProviders).perform();
 
-        const providers = [...legacyProviders, ...serviceStorageProviders]
-            .sort(this.providerSorter);
-        this.addonProviders = providers;
+        serviceStorageProviders.sort(this.providerSorter);
```

Comment:
Hmmm. They're not just storage providers, though, are they? What was I thinking here? We can fix this sometime in the future.